uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrix.php

```diff
@@ -83,17 +83,17 @@ private function shouldUseManagerMatrixForCurrentUser($leaveRequestContactID) {
    */
   private function getStaffStatusMatrix() {
     $leaveRequestStatuses = self::getLeaveRequestStatuses();
+    $matrix = [];
 
-    return [
-      $leaveRequestStatuses['waiting_approval'] => [
-        $leaveRequestStatuses['waiting_approval'], $leaveRequestStatuses['cancelled']
-      ],
-      $leaveRequestStatuses['more_information_requested'] => [
-        $leaveRequestStatuses['waiting_approval'], $leaveRequestStatuses['cancelled']
-      ],
-      $leaveRequestStatuses['approved'] => [$leaveRequestStatuses['cancelled']],
-      NULL => [$leaveRequestStatuses['waiting_approval']]
+    $matrix[$leaveRequestStatuses['waiting_approval']] = [
+      $leaveRequestStatuses['waiting_approval'],
+      $leaveRequestStatuses['cancelled']
     ];
+    $matrix[$leaveRequestStatuses['more_information_requested']] = $matrix[$leaveRequestStatuses['waiting_approval']];
```

Comment:
Please add an empty line between each assignment here (and in the next method too)